code/game/objects/items/trash.dm

```diff
@@ -7,6 +7,16 @@
 	w_class = WEIGHT_CLASS_TINY
 	resistance_flags = FLAMMABLE
 
+/obj/item/trash/Initialize()

```

Comment:
You may want to consider using the mapload arg to skip trash that is placed by the map, unless that is ok with you.

Reply:
u should put in mapload regardless 8)

Reply:
The janitor has to clean up whatever starts on the level. The exception would be maint, which is probably where most roundstart trash begins. I wonder if there's a maint areas define.

Reply:
What will adding mapload do? @ExcessiveUseOfCobblestone

Reply:
I'm not cobby but it defines the arg for consistency even if you don't use it. Can't see another reason

Reply:
yes